x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/sortable_list.tsx

```diff
@@ -11,27 +11,17 @@ import {
   EuiDroppableProps,
   EuiDragDropContext,
   EuiDroppable,
-  useEuiTheme,
 } from '@elastic/eui';
-import { css } from '@emotion/react';
 
 interface SortableListProps {
   onDragItem: DragDropContextProps['onDragEnd'];
   children: EuiDroppableProps['children'];
 }
 
 export const SortableList = ({ onDragItem, children }: SortableListProps) => {
-  const { euiTheme } = useEuiTheme();
-
   return (
     <EuiDragDropContext onDragEnd={onDragItem}>
-      <EuiDroppable
-        droppableId="droppable-area"
-        css={css`
-          background-color: ${euiTheme.colors.backgroundTransparent};
-          margin-bottom: ${euiTheme.size.s};
-        `}
-      >
+      <EuiDroppable droppableId="droppable-area" spacing="m">
```

Comment:
As we discussed offline, this change should be reverted and the margin-bottom should be applied to avoid having the add processor button stick to the processor list.